test/conformance/resources/rabbitmqcluster/rabbitmqcluster.yaml

```diff
@@ -12,3 +12,13 @@ spec:
     requests:
       cpu: "100m"
       memory: "1Gi"
+  override:
+    statefulSet:
+      spec:
+        template:
+          spec:
+            containers:
+            - name: rabbitmq
+              env:
+              - name: ERL_MAX_PORTS
```

Comment:
Is this change still needed?

Reply:
mentioned it today during retro - https://github.com/rabbitmq/cluster-operator/issues/959

Reply:
But we're not experiencing a crash loop with the node not starting though right? Our RMQ version is also newer (3.10) I believe so maybe that's why we're bypassing the issue?

Reply:
it doesn't depend on RMQ version. it's a combination of host setup and memory limit. When Erlang starts it tries to allocate memory stuctures for all available FDs. For example:

```
cat /proc/sys/fs/file-max 
9223372036854775807
```

So the fact that it works locally and on GH is a pure luck

Reply:
I can just barely grok what's happening with Erlang here. I understand that with a too high FD limit, Erlang can cause RMQ broker to OOM as it tries to allocate something per FD. 

But it does look like this change isn't directly tied to the DLQ flake we're seeing in conformance tests. Maybe we can omit this change for this PR to unblock it and then open something separate to discuss this? We also have RMQClusters defined elsewhere in docs and setup instructions so we'll likely need a broader change than just the test clusters.

Reply:
>> I can just barely grok what's happening with Erlang here. I understand that with a too high FD limit, Erlang can cause RMQ broker to OOM as it tries to allocate something per FD.

you got it completely right. we have to limit the subset erlang sees because we don't control test environments. it's like restricting max file size while doing uploads handler, etc. 

Having really high fd limit is not a problem of course. The problem is this particular software pattern of preallocating stuff.

It's of cpurse possible for me to have this change in a separate PR. However, on my system tests can't be run without capping ERL_MAX_PRTS. So that's why it goes as a package.